### DISCUSSION TITLE
How to increase scroll speed?

### DISCUSSION THREAD

**Discussion by D37** [noah-nuebling/mac-mouse-fix#1138] [Questions]
I put the scroll speed to "High", but it's still too slow. How do I increase it?

**Comment by noah-nuebling** [comment]
Hi Ryan, I just read your email,
Thanks for your feedback, I’m really glad you’re excited about the app :)

Scrolling speed is limited right now. I do plan on turning the Scroll Speed into a slider instead of just “Low, Medium, High” and that slider would let you make scrolling a little faster than you can right now. However, I’m not sure when that’s coming, as there are a lot of things I want to work on for the app, and even just implementing the slider options really well could require quite a bit of work.

In the meantime, you could try setting the scroll speed to “macOS” and then seeing if you can make the scrolling fast enough for your tastes in “System Settings > Mouse”

There’s also the “Swift Scroll” modifier, that lets you scroll faster, but you’d have to hold a mouse button or keyboard modifier to activate it.

I hope this helps a bit or clears things up! Let me know what you think about this or if you have any more questions.

Best
Noah

**Reply by D37** [reply]
Hey Noah, thank you for your reply. 

The idea of a slider to make the scrolling fast faster is wonderful! I think I will wait for that feature. 

I tried using the macOS setting, but the scroll speed is not that fast. In fact, it’s slower than the highest setting in your app. 

Thank you for your development on this app!

**Comment by keesiemeijer** [comment]
Hi Noah

I also like the idea of a slider. For me, the scroll speed could just be a little bit higher than the medium setting. Using the “macOS” setting makes scrolling a whole lot worse.

Thanks for a great app.
